Tribler/community/triblerchain/community.py

```diff
@@ -237,6 +238,10 @@ def get_graph(self, public_key=None, neighbor_level=1):
 
         return_nodes = nodes.values()
         return_edges = []
+
+        for node in return_nodes:
+            node.update({"score": calculate_score(node)})
```

Comment:
Calling update is not necessary here, why not just `node["score"] = calculate_score(node)`?